src/main/scala/Keyboard.scala

```diff
@@ -0,0 +1,37 @@
+object Keyboard extends App {
+  import scala.io.Source
+  import scala.io.StdIn._
+  println("Enter number of character strings to read")
+  val wordAmount = readInt
+  var stringsArray = new Array[String](wordAmount)
```

Comment:
var stringsArray = new Array[String](wordAmount) here I think you wanted to create an array of the length read, but what it does if im not wrong is that it creates the array with that element inside rather than of that length